proposals/0336-distributed-actor-isolation.md

```diff
@@ -1643,7 +1643,7 @@ For example, the [distributed actor cluster implementation](https://github.com/a
 + }
 ```
 
-Under implicit `distributed func` rules, it is impossible to know if this function is possible to be invoked remotely. And if it were so, it could be a potential exploitation vector. Of course transports do and will implement their own authentication and authorization mechanisms, however nevertheless the inability to know if we just added a remotely invokable endpoint is worrying.
+Under implicit `distributed func` rules, it is impossible to know if this function is possible to be invoked remotely. And if it were so, it could be a potential exploitation vector. Of course transports do and will implement their own authentication and authorization mechanisms, however nevertheless the inability to know if we just added a remotely invocable endpoint is worrying.
```

Comment:
Both "invokable" and "invocable" are extant spellings, so one would defer to the author.